### DISCUSSION TITLE
RSL Helper не работает

### DISCUSSION THREAD

**Discussion by 4e6yPaIII** [FarbstoffRSL/RSL-Helper#476] [Q&A]
Установил, обновил, запустил програму.
Кликер автоматом запустил плариум и Raid.
Но не может присоединиться к аккаунту. Просит поиграть в игру и т.п. Уже закрыл весь дневной контент кликер так и не увидел данные об аккаунте
![eOyvlYd6PwA](https://user-images.githubusercontent.com/122114915/211064581-30cc94d7-0a28-4020-bc09-2d476240b75d.jpg)

**Comment by Rimmanata** [comment]
У меня почему то не видит мой ID...в чём может быть проблема???
